### DISCUSSION TITLE
How to hide the container and only show the cropper?

### DISCUSSION THREAD

**Discussion by laduniestu** [ValentinH/react-easy-crop#493] [Q&A] [answered]
this is what i got
![Screenshot 2023-10-23 at 01 13 09](https://github.com/ValentinH/react-easy-crop/assets/44757637/dd02b923-ff8c-4168-9d34-4fc94bf719c7)

this is what i want
![Screenshot 2023-10-23 at 01 19 41](https://github.com/ValentinH/react-easy-crop/assets/44757637/ddb3fd4d-1dd8-4cd2-a593-54e9d51dc618)

**Comment by ValentinH** [comment, marked as answer]
You might get what you want with the `objectFit="cover"` property

**Reply by laduniestu** [reply]
Wow awesome, Thanks !

**Reply by laduniestu** [reply]
is it possible if i want to set the container just 5px outside the cropper? @ValentinH

**Reply by ValentinH** [reply]
This is a workaround but you can probably override the `.reactEasyCrop_Container` class and set `overflow: visible`. Then you wrap the Cropper component in a div where you set `overflow: hidden` and add a 5px padding.
